flink-end-to-end-tests/test-scripts/test-runner-common.sh

```diff
@@ -64,9 +64,7 @@ function run_test {
         fi
     fi
 
-    if [[ ${exit_code} == 0 ]]; then
-        cleanup
-    else
+    if [[ ${exit_code} != 0 ]]; then
```

Comment:
if there is only `exit "${exit_code}"` in there I think we can completely drop it